packages/wrangler/src/__tests__/jest.setup.ts

```diff
@@ -9,6 +9,11 @@ jest.mock("undici", () => {
     fetch: jest.requireActual("jest-fetch-mock"),
   };
 });
+// Outside of the Sentry tests themselves, we mock Sentry to ensure that it doesn't actually send any data and
+// that it doesn't interfere with the rest of the tests.
+jest.mock("../sentry");
+
+jest.setMock("node-fetch", () => jest.requireActual("jest-fetch-mock"));
```

Comment:
Modified this from `.mock()` to `.setMock` which fixed an issue where the fetch was throwing an `isntanceof` Type Error in the logout test using the jest-fetch-mock

Reply:
![Uploading Screen Shot 2022-02-07 at 11.38.27 AM.png…]()

Reply:
<img width="1599" alt="Screen Shot 2022-02-07 at 11 38 27 AM" src="https://user-images.githubusercontent.com/27247160/152850247-8fbfbab5-f6ef-46a1-b26c-3118651f21ca.png">

Reply:
This could need to be changed the mock to handle `undici` and possible replace & remove `jest-fetch-mock` if it cannot handle the `undici` fetch implementation.

Reply:
I thought we had already moved over to undici and were no longer using `node-fetch`??

Reply:
Might have been a rebase that I did and that got lost in the jungle.